src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/IntegrationTests/ResultsServiceTests.cs

```diff
@@ -46,6 +46,63 @@ public void GetAllTestResults_GetsAllTestResults()
             }
         }
 
+        [Fact]
+        public void GetTestResultsForWeek_ShouldReturnOnlyResultsForSecondWeek()
```

Comment:
it should beGetTestResultsForSecondWeek_ShouldReturnOnlyResultsForSecondWeek

Reply:
we should have one big test that adds results and userTestCorrectAnswers for each week and checks every week points